src/app/debug/table/table-settings-modal/table-settings-modal.component.html

```diff
@@ -113,7 +113,7 @@ <h5>Report table</h5>
       <button title="Revert changes" type="button" class="btn btn-info" (click)="loadSettings()"><i
         class="fa fa-undo"></i></button>
       <button data-cy-settings='saveChanges' title="Save changes" type="submit" id="saveTableSettings"
-              class="btn btn-info" (click)="modal.dismiss()"><i class="fa fa-save"></i></button>
+              class="btn btn-info" (click)="modal.dismiss(); saveSettings()"><i class="fa fa-save"></i></button>
```

Comment:
Small clean improvement: move click logic to its own function (prevent multiple statements in template)

Reply:
Fixed